src/gt4py/next/ffront/func_to_past.py

```diff
@@ -162,6 +162,19 @@ def visit_BinOp(self, node: ast.BinOp, **kwargs: Any) -> past.BinOp:
     def visit_Name(self, node: ast.Name) -> past.Name:
         return past.Name(id=node.id, location=self.get_location(node))
 
+    def visit_Attribute(self, node: ast.Attribute) -> past.AttributeExpr:
+        attr_type = (
+            type_translation.from_value(self.closure_vars[node.attr])
+            if node.attr in self.closure_vars
+            else None
+        )
+        return past.AttributeExpr(
+            attr=node.attr,
```

Comment:
probably
```suggestion
            attr=self.visit(node.attr),
```

Reply:
`node.attr`is a string. Do we need a visit for that?

Reply:
makes sense to not visit the `str`, I thought it's an `ast.Name`.